File: python-sdk/src/boxsdk/models/shared/sessionterminationmessage.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils


@dataclass_json
@dataclasses.dataclass
class SessionTerminationMessage:
    r"""SessionTerminationMessage
    A message informing about the
    termination job status
    """
    
    message: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('message') }})
    

File: ts-sdk/src/sdk/models/shared/sessionterminationmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// SessionTerminationMessage
/** 
 * A message informing about the
 * termination job status
**/
export class SessionTerminationMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=message" })
  message?: string;
}
